### ISSUE TITLE
Duplicate version 1.3

### ISSUE THREAD

**Issue by darksidemilk** [OneGet/oneget#429]
# The Problem

The latest version of packagemanagement appears to have been released twice in the powershell gallery. Once as 1.3 and then again as 1.3.0. 

However, because both are listed and are both equal to 1.3, any versioning automations someone may have can end up catching both versions as the latest and mess up automatic upgrading.

## A bit more detail

This is happening to me while using Inedo Proget which I use to host an internal Powershell Repository that connects to powershell gallery so that I can install internal and external packages all from one repository. With the current versions being the same it's causing some weird behaviors.
i.e.
```
inmo packagemanagement -force;
```
This should simply install the latest version from the registered repo right?
Instead I'm getting an error saying there are multiple versions found.

## Suggested Solutions

If either the 1.3 version could become 'unlisted' in the powershell gallery, or the 1.3.0 were to be re-released/published as 1.3.0.1 or 1.3.1 

# TL;DR

The duplicate 1.3 versions published in powershell gallery are causing issues with custom automations that are based on package management versioning standards. This can be fixed by either unlisting one of the duplicates in powershell gallery, or re-releasing 1.3.0 as 1.3.0.1 would fix the problem.

**Comment by darksidemilk** [comment]
Also, please and thank you. Don't want to sound all demanding or anything like that.

**Comment by darksidemilk** [comment]
I just went to the powershell gallery to add links to each published version and found that 1.3 and 1.3.0 were both unlisted now, so maybe someone caught this before I put it here.

**Comment by darksidemilk** [comment]
It seems to be a somewhat intermittent issue. Can't recreate it 100% of the time. May be related to another issue when the chocolatey for business package manager license expires. That sometimes seems to effect powershell package management in weird ways.

**Comment by darksidemilk** [comment]
It appears 1.3.1 has been released. 
https://www.powershellgallery.com/packages/PackageManagement/1.3.1
For this I say thank you good sirs.

**Comment by edyoung** [comment]
I'm going to close this for now, because as far as I can tell there isn't a duplicate version of packagemanagement 1.3. If you or anyone else has a more reliable repro case, please feel free to reopen this with details or file a new issue. Hope that makes sense.
